### PR TITLE
refactor: use whiskers

### PR DESCRIPTION
**Pull request by uncenter**
Adds [Whiskers](https://github.com/catppuccin/toolbox/tree/main/whiskers), Catppuccin's in-house templating tool, to build the themes. Instead of the previous Deno build script, edit the `spyder.tera` file and then run `just build` to build the output themes. You will need Whiskers (linked above) and [Just](https://github.com/casey/just) installed.

### CODE REVIEW

File: themes/all.ini [hunk 1]
```diff
@@ -16,11 +16,11 @@ catppuccin/latte/comment = ('#8c8fa1', False, True)
 catppuccin/latte/string = ('#40a02b', False, False)
 catppuccin/latte/number = ('#fe640b', False, False)
 catppuccin/latte/instance = ('#d20f39', False, True)
-catppuccin/frappe/name = Catppuccin Frappe
+catppuccin/frappe/name = Catppuccin Frappé
```
Comment (uncenter) [inline themes/all.ini]: Can revert if this isn't supported.

File: themes/all.ini [hunk 2]
```diff
@@ -2,7 +2,7 @@ catppuccin/latte/name = Catppuccin Latte
 catppuccin/latte/background = #eff1f5
 catppuccin/latte/currentline = #ccd0da
 catppuccin/latte/currentcell = #eff1f5
-catppuccin/latte/occurrence = #b0c7f5
```
Comment (uncenter) [inline themes/all.ini:5]: All changes to `occurence` as seen below are due to the colors for it not being valid Catppuccin colors. I used the closest color to it though, which is `surface1`.

File: themes/all.ini (hunk 1 above)
Comment (sgoudham) [reply themes/all.ini]: I really don't want to install Spyder to test this lol, I'd prefer the revert if that's alright.

**Review by sgoudham — CHANGES_REQUESTED** [review]
I don't think @meme8383 uses Spyder anymore so I'm happy to review this one. Just one comment from me.

**Review by sgoudham — APPROVED** [review]

**Pull request merged**